ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -640,7 +640,7 @@ export const DatasetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), wi
         this.setState(({domainValueSetIsLoading, domainValueSetLookup}) => {
           const newLoading = new Set(domainValueSetIsLoading);
           domainsToLoad.forEach((d) => {
-            if (!domainValueSetIsLoading.has(d) && !domainValueSetLookup.has(d)) {
+            if (d && !domainValueSetIsLoading.has(d) && !domainValueSetLookup.has(d)) {
```

Comment:
Sometimes these are null for whatever reason. Waiting to see if puppeteer still dies.